.github/workflows/enforce-labels.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     steps:
     - uses: yogevbd/enforce-label-action@2.1.0
       with:
-        REQUIRED_LABELS_ANY: "verified,trivial,needs-verification"
-        REQUIRED_LABELS_ANY_DESCRIPTION: "PR must be labeled with 'trivial', 'needs-verification', or 'verified'"
-        BANNED_LABELS: "do-not-merge,duplicate,invalid,wontfix"
+        REQUIRED_LABELS_ANY: "verified,trivial,needs-verification,zero impact"
```

Comment:
Mega-nit: `needs-verification` has a hyphen, `zero impact` doesn't, can we adapt one of them?

Reply:
Everything with a hyphen sounds good to me!